mne/viz/raw.py

```diff
@@ -97,12 +97,15 @@ def _pick_bad_channels(event, params):
     _plot_update_raw_proj(params, None)
 
 
+_RAW_CLIP_DEF = 3.
```

Comment:
Should this maybe be a `config` setting? Or wouldn't that make any sense?

Reply:
No this is just for internal convenience, not user setting